librz/include/rz_endian.h

```diff
@@ -128,7 +128,7 @@ static inline void rz_write_at_be64(void *dest, ut64 val, size_t offset) {
 }
 
 static inline ut128 rz_read_be128(const void *src) {
-	ut128 val = { 0 };
+	ut128 val = { 0, 0 };
 	val.High = rz_read_be64(src);
 	val.Low = rz_read_at_be64(src, sizeof(ut64));
```

Comment:
since the `High` and `Low` values are set immediately after, we don't need to initialize this.